src/molecules/side-menu.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './side-menu.module.css';
+
+const SideMenu = ({ isOpen, closeMenu, subjects }) => {
+  return (
+    <>
+      {isOpen && <div className={styles.overlay} onClick={closeMenu}></div>}
+      <div className={styles.sideMenu} data-open={isOpen ? 'true' : 'false'}>
+        <div className={styles.topBar}>
+          <button className={styles.closeButton} onClick={closeMenu}>×</button>
+        </div>
+        <button className={styles.homeButton}>ホーム</button>
```

Comment:
```suggestion
        <button className={styles.menuItemButton}>ホーム</button>
```
`homeButton` というスタイルは無いようですが, おそらくこういう意図であったのかと思います.